assets/.browserslistrc

```diff
@@ -0,0 +1,10 @@
+# DUP
+ChromeAndroid 61
+# Sectional
+Firefox 126
```

Comment:
This should be `Firefox 90` since that seems to be what the Solari units have. Maybe we should be explicit in this file about which vendors' hardware we're referring to, since there is not a 1-to-1 mapping of "screen app" to hardware specs or vendors. For example we could have:

```
# Sectional (Solari)
Firefox 90
# Sectional (GDS)
Firefox 126
```

Then someday when we retire the Solaris, we know we should delete that line, but not the other one.